src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    protected Task(String description) {
+        this.description = description;
+        this.isDone = false;
```

Comment:
Coding standard suggests avoiding unnecessary use of `this` with fields. (`isDone` here, in particular.)
https://se-education.org/guides/conventions/java/index.html#variables